tracing-attributes/src/lib.rs

```diff
@@ -878,7 +878,7 @@ enum RecordType {
 
 impl RecordType {
     /// Array of primitive types which should be recorded as [RecordType::Value].
-    const TYPES_FOR_VALUE: [&'static str; 23] = [
+    const TYPES_FOR_VALUE: [&'static str; 25] = [
```

Comment:
take it or leave it: this array could probably be a slice, if we want to avoid having to update the length every time it changes:

```suggestion
    const TYPES_FOR_VALUE: &[&'static str] = &[
```